Applications/AdminUi/apps/admin_ui/lib/screens/splash_screen.dart

```diff
@@ -45,18 +45,22 @@ class _SplashScreenState extends State<SplashScreen> {
     await GetIt.I.allReady();
     final sp = await SharedPreferences.getInstance();
 
-    if (!sp.containsKey('api_key')) return _navigate('/login');
+    if (!sp.containsKey('api_key')) return _navigateToLogin();
```

Comment:
after login the page you wanted to go is completely lost, right?

Reply:
I see what you mean.